src/MTGCore/ViewModels/SearchFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace MTGCore.ViewModels
+{
+    public class SearchFilter
+    {
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string Rarity { get; set; }
+        public string Set { get; set; }
+        public string Price { get; set; }
+    }
+
+    public class SearchFilterWithColours : SearchFilter
+    {
+        public string ColoursOption { get; set; }
+        public List<string> Colours { get; set; }
+    }
```

Comment:
I don't see any benefit in this?

Why not have all the properties in the same class? Also, it might be worth storing the colours in an enum just to restrict which values can be used?

Reply:
well originally only the basic strings were implemented and i wanted to do the colour stuff later so i just put it in a derived class.